### PR TITLE
Broadcast signed transaction function - Fixes #124

### LINKED ISSUE #124: Broadcast signed transaction function
To send a whole transaction object to the network.

### PR DESCRIPTION
**Pull request by Tosch110**
Fixes #124 

Adds the broadcastSignedTransaction function to the API method.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 <script src="dist/lisk-js.js"></script>
 <script>
-    "use strict";
+
```
Comment (MaciejBaj) [inline index.html]: remove script tag

File: test/api/liskApi.js [hunk 2]
```diff
@@ -779,4 +779,22 @@ describe('Lisk.api()', function () {
 			});
 		});
 	});
+
+	describe('#broadcastSignedTransaction', function () {
+
+		it('should be able to broadcast a finished and signed transaction', function (done) {
+
+			var LSKAPI = lisk.api({testnet: true});
+			var amount      = 0.001 * Math.pow(10, 8);
```
Comment (MaciejBaj) [inline test/api/liskApi.js]: formatting

File: docs/example/api.html [hunk 3]
```diff
@@ -406,6 +407,34 @@ <h3 id="functions_listVoters">listVoters</h3>
...
+                <div class="panel panel-default">
+                    <div class="panel-body">
+                        <div class="rows">
+                            <div class="col-md-12">
+                                <h3>Example <small>Create a signed transaction, use broadcastSignedTransaction to send it to the network</small></h3>
+                                <figure class="highlight">
+                                    <pre>
+                                        <code class="language-html" data-lang="html">
+var LSK = lisk.api({testnet: true});
+
```
Comment (MaciejBaj) [inline docs/example/api.html]: looks wierd here

Comment (Tosch110) [inline docs/example/api.html]: Yes, the html formating within <code> tags looks weird. But that is necessary for the display.

**Pull request merged**